Git-commit: 27d2e7004b9b9162b8f8e5cddd67886c01f2dbf7
From: Dom Cobley <popcornmix@gmail.com>
Date: Wed, 6 Jan 2021 18:16:29 +0000
Subject: [PATCH] bcm2711-rpi.dtsi: Bump hdmi audio dma panic priority to max
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

Set panic priority to 15 and leave normal priority at 0

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2711-rpi.dtsi b/arch/arm/boot/dts/bcm2711-rpi.dtsi
index 3dbfd601ca39..8053ef54acc8 100644
--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -179,7 +179,7 @@ &pixelvalve4 {
 };
 
 &hdmi0 {
-	dmas = <&dma (10|(1<<27)|(1<<24))>;
+	dmas = <&dma (10|(1<<27)|(1<<24)|(0<<16)|(15<<20))>;
 	status = "disabled";
 };
 
@@ -188,7 +188,7 @@ &ddc0 {
 };
 
 &hdmi1 {
-	dmas = <&dma (17|(1<<27)|(1<<24))>;
+	dmas = <&dma (17|(1<<27)|(1<<24)|(0<<16)|(15<<20))>;
 	status = "disabled";
 };
 
-- 
2.31.1

